napari/layers/utils/plane_manager.py

```diff
@@ -77,3 +73,59 @@ def from_points(cls, a, b, c):
         plane_normal = np.cross(ab, ac)
         plane_position = np.mean(abc, axis=0)
         return cls(position=plane_position, normal=plane_normal)
+
+
+class PlaneList(EventedList):
+    """
+    A list of planes providing some utility methods
+    """
+
+    def as_array(self):
+        """
+        return a (N, 2, 3) array representing the planes
+        """
+        arrays = []
+        for plane in self:
+            if plane.enabled:
+                arrays.append(np.stack([plane.position, plane.normal]))
```

Comment:
this is nice!

Reply:
To answer you top comment: `plane.enabled` works as you might expect! Since I connected the whole `EventedList` to the update method, this will fire for any `PlaneManager` event, including changing `plane.enabled`. Try to run the example from ipython, then:
```python
volume_layer.clipping_planes[0].enabled = False
```

Reply:
doh! Should've checked - that's awesome :)